src/control/cmd/daos/filesystem.go

```diff
@@ -318,16 +323,19 @@ func (cmd *fsGetAttrCmd) Execute(_ []string) error {
 			return cmd.OutputJSON(jsonAttrs, nil)
 		} else {
 			jsonAttrs := &struct {
+				OID       string `json:"oid"`
```

Comment:
this is a little odd in that the directory has object attributes and directory attributes where the object attributes are under "object" but these file ones are at top level.   Maybe we should consider uniformity.  Anyway, I worked around it in node_local_test but it's a little weird.

Reply:
well for directory there are 2 different things, one the gives the object class of the directory itself  and the other that describes what object classes would be used for files  and directories that would be created in that dir.
obviously for files we cannot create subfiles/subdirs, so it would not make sense to me to have the same map there. so I would think it would be weird if it's actually uniform.

Reply:
happy to leave it as is.